reference/entry.json

```diff
@@ -3893,7 +3893,7 @@
             }
           }
         },
-        "description": "Creates a news article.",
+        "description": "Creates a news article.\n\n<!-- theme: warning -->\n\n> For backward compatibility with older API clients, this path is also available at `/news_articles`. However the preferred path is `/news-articles`.",
```

Comment:
I wonder if this should be a warning? Maybe an info is a better level?

Reply:
Good point, I was going back and forth between `warning` and `success` yesterday, but `info` is actually perfect 😄